lib/db_helper/tasks.rb

```diff
@@ -11,50 +11,52 @@
 namespace :db do
   desc "Copy and import a remote database to a local database"
   task :import_from_remote do
-    begin
-      sources = YAML::load(File.open(File.join(ENV['RAILS_ROOT'] || '.', File.join('config','database_import.yml'))))
-      default = sources.delete('default') || {:db => {}}
+    root = ENV['RAILS_ROOT'] || '.'
+    path = File.join(root, 'config', 'db_helper_import_from_remote.yml')
+    sources = YAML.load(File.open(path))
+    default = sources.delete('default') || {db: {}}
```

Comment:
Space inside { missing.<br>Space inside } missing.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.